README.md

```diff
@@ -5,6 +5,15 @@ For more information, questions, or just the use of the hosted version, you can
 ![Example image](https://i.imgur.com/iAP1rMs.png)
 
 ## Usage
+The tracked repositories can be configured in multiple ways.
+
+### Bot-command
+Open a conversation with the bot and type `!github <username>`.
```

Comment:
I'd word this as something more like "Send the message `!github <username>` to the bot" instead, "open a conversation and type" sounds far too technical for what it is